actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -29,15 +35,15 @@ def cache_collection_render(instrumentation_payload, view, template)
         # Extract the items for the keys that are not found
         # Set the uncached values to instance variable @collection
         # which is used by the caller
-        @collection = keyed_collection.reject { |key, _| cached_partials.key?(key) }.values
+        collection = keyed_collection.reject { |key, _| cached_partials.key?(key) }.values
 
         # If all elements are already in cache then
         # rendered partials will be an empty array
         #
         # If the cache is missing elements then
         # the block will be called against the remaining items
         # in the @collection.
```

Comment:
Looks like the documentation is now slightly off here and elsewhere in the file. Could update or maybe scrap? Think it blurs more than helps?

Reply:
I'll remove it.  Now that the caller is required to use the collection that's yielded to the block, I don't think we need this documentation (the ivar isn't mutated anymore).